### PR TITLE
Release notes for Discovery API

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Release notes for the public preview of the dbt Cloud Discovery API

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Latest environment state in the dbt Cloud Discovery API"
+id: "discovery-api-public-preview"
+description: "Public preview of the dbt Cloud Discovery API is now available."
+sidebar_label: "New: Latest environment state in the Discovery API"
+tags: [May-24-2023, API]
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md:7]: Can you add a `sidebar_position: 5` to the header so it shows up above The [other one](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/docs/docs/dbt-versions/release-notes/08-May-2023/run-history-endpoint.md). THis is a docusaurus thing and I think it's written down. I can verify if it's in our docs process later!

**Review by runleonarun — DISMISSED** [review]
LGTM one minor comment!

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md (hunk 1 above)
Comment (nghi-ly) [reply website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md:7]: totally! didn't know about `sidebar_position` . thank you!

File: website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Latest environment state in the dbt Cloud Discovery API"
+id: "discovery-api-public-preview"
+description: "Public preview of the dbt Cloud Discovery API is now available."
+sidebar_label: "New: Latest environment state in the Discovery API"
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/08-May-2023/discovery-api-public-preview.md:5]:
```suggestion
sidebar_label: "New: Latest environment state in the Discovery API"
sidebar_position: 5
```

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**